app/views/account_requests/new.html.erb

```diff
@@ -28,6 +28,7 @@
       <div class='card-text' id='partner_info'>
         <p class='p-2'>Are you a current partner to diaper and/or period supply banks? If so please go <a href='https://humanessentials.app/users/sign_in'>here</a> to login.</p>
         <p class='p-2'>If you are wishing to receive diapers/period supplies and partner to a local diaper/period supply bank please contact your local bank. They are the only ones able create your account.</p>
+        <p class='p-2'>If you are looking for an essentials banks that distributes diapers, please refer to the <a href='https://nationaldiaperbanknetwork.org/member-directory/'>NDBN member directory</a>. If you are, instead, looking for an essentials bank that deals in period supplies, please refer to the <a href='https://allianceforperiodsupplies.org/allied-programs/'>Alliance for Period Supplies directory</a>.</p>
```

Comment:
There was a typo in the original text  from the issue -- the first "essentials banks"  should be "essential bank" .